src/main/java/com/google/sps/data/Session.java

```diff
@@ -0,0 +1,126 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+import java.util.Optional;
+import com.google.sps.data.EntityConstants;
+import java.util.List;
+
+/** Class that represents a session. */
+public class Session implements SessionInterface {
+
+  // The id used to identify the session. 
+  private final String sessionId;
+
+  // The screen name of the user with the controller. 
+  private Optional<String> screenNameOfController;
+
+  // The ip of the VM assigned to this session. 
+  private Optional<String> ipOfVM;
+
+  // List of attendees in this session.
+  private List<String> listOfAttendees; 
```

Comment:
hmm im not sure of it should be List<Attendees> listOfAttendees or just considering the listOfScreenNamesOfAttendees; 
Additionally I imagine that the listOfAttendees would initially be null at first, so how do you add names/attendees?

Reply:
The datastore manager will create the list as empty initially, then update once attendees start to join.